tests/integration/test_vulnerability_detector/test_general_settings/test_interval.py

```diff
@@ -11,10 +11,16 @@
        is working correctly. This option is located in its corresponding section of
        the `ossec.conf` file and allows to define the time between scans.
 
+<<<<<<< HEAD
```

Comment:
Conflicts have not been properly resolved

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/2649/commits/71e4587f39df3f93b1e16f8387b2c14fb5bc217b